app/cdap/components/ColumnInsights/Components/ColumnDetails/index.tsx

```diff
@@ -0,0 +1,194 @@
+  };
+
+  const checkForInvalidInput = (renamedString: string) => {
+    if (renamedString === '' || !/^\w+$/.test(renamedString)) {
+      setErrorMessage({
+        hasError: true,
+        message: displayMessage.invalidMessage,
+      });
+    } else if (columnHeaderList.includes(renamedString) && columnName !== renamedString) {
+      setErrorMessage({
+        hasError: true,
+        message: displayMessage.matchedColumnMessage,
+      });
+    } else {
+      setErrorMessage({
+        hasError: false,
+        message: '',
+      });
+    }
+  };
+
+  const handleDataTypeChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setDataTypeValue(e.target.value);
+    dataTypeHandler(e.target.value);
+  };
+
+  const editHandler = () => {
+    setCanEdit(true);
+  };
+
+  const onChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    checkForInvalidInput(e.target.value);
+    setInputValue(e.target.value);
+  };
+
+  const onEnter = (e: React.KeyboardEvent<HTMLElement>) => {
+    if (
+      (e.target as HTMLInputElement).value !== columnName &&
+      !errorMessage.hasError &&
+      e.keyCode === 13
+    ) {
+      renameColumnNameHandler(columnName, (e.target as HTMLInputElement).value);
+      setCanEdit(false);
+    } else {
+      setCanEdit(true);
+    }
+  };
+
+  return (
+    <ColumnDetailsContainer data-testid="column-details-parent">
+      <CustomizedColumnNameEditBox>
+        {canEdit ? (
+          <input
+            value={inputValue}
+            onChange={(e) => onChangeHandler(e)}
+            onKeyDown={(e) => onEnter(e)}
+            data-testid="column-name-edit-input"
+            autoFocus
+          />
+        ) : (
+          <RenderLabel fontSize={16} dataTestId={'column-name'}>
+            <> {inputValue}</>
+          </RenderLabel>
+        )}
+        <CustomizedIconButton onClick={editHandler} aria-label="edit-icon" data-testid="edit-icon">
+          <EditIcon />
+        </CustomizedIconButton>
+      </CustomizedColumnNameEditBox>
+      {errorMessage.hasError && (
+        <RenderLabel fontSize={14} color={`${red[600]}`} dataTestId={'invalid-text'}>
+          <> {T.translate(errorMessage.message).toString()}</>
+        </RenderLabel>
+      )}
+
+      <InputSelect
+        defaultValue={defaultValueProvided[0]?.value}
+        value={dataTypeValue}
+        onChange={(e: React.ChangeEvent<HTMLInputElement>) => handleDataTypeChange(e)}
+        options={DATATYPE_OPTIONS}
+        fullWidth={false}
+        type={'column-insights'}
+      />
+      <ColumnInsightsDetailsBox>
+        <ColumnInsightsDetailsCountBox>
+          <RenderLabel fontSize={14}>
+            <>
+              {T.translate(`${PREFIX}.characterCount`).toString()} {characterCount}
+            </>
+          </RenderLabel>
+
+          <RenderLabel fontSize={14}>
+            <>
+              {T.translate(`${PREFIX}.distinct`).toString()} {distinctValues}
```

Comment:
Here too, for `distinctValues`

Reply:
Fixed it .
Changed it to  <>{T.translate(`${PREFIX}.distinct`, { distinctValues }).toString()}</>